x-pack/plugins/observability_solution/slo/public/pages/slos/components/grouped_slos/group_list_view.tsx

```diff
@@ -54,8 +54,8 @@ export function GroupListView({
   summary,
   filters,
 }: Props) {
-  const groupQuery = `"${groupBy}": "${group}"`;
-  const query = kqlQuery ? `"${groupQuery}) and ${kqlQuery}` : groupQuery;
+  const query = kqlQuery ? `"${groupBy}": (${group}) and ${kqlQuery}` : `"${groupBy}": ${group}`;
```

Comment:
@kdelemme I reverted to what I had before, cause it was breaking the kql query. I tried to remove an extra parenthesis I saw you added here ``"${groupQuery})`, but it didn't work. So I used the previous version I had.

Reply:
I see now, there was an extra `""`. I'll use what you had, without the extra `"` and `)`